mautic/mautic
# Chaining purify method to ensure safe rendering of translation keys

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🟢 
| New feature/enhancement? (use the a.x branch)      | 🔴
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #14237 


## Description



|#| Before                                 | After
|-| -------------------------------------- | ---
|1| <img width="643" alt="Before" src="https://github.com/user-attachments/assets/85aecea6-1e86-4960-b8ac-6da1d0a3a9fd">                                        | <img width="654" alt="After" src="https://github.com/user-attachments/assets/923fb836-f173-4ef2-a346-cd9f47aa3f60">
|2| <img width="633" alt="2 - Before" src="https://github.com/user-attachments/assets/b0325512-5087-4c2b-a435-fba2dd41db35"> | <img width="629" alt="2 - After" src="https://github.com/user-attachments/assets/ab98f780-49dd-41bb-89f4-6b615279e6fc">


---
### 📋 Steps to test this PR:

1. Open this PR on Gitpod or pull it down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. For Contact (Refer to point 1 from before and after)
   1. Create a Contact
   2. Create a Segment; add the above-created contact to this segment
   3. Create a Campaign with the above-created segment 
   4. Add following Actions,
      1. Add Do No Contact
      2. Send email
   5. Save the Campaign
   6. Execute it.
   7. Browse the contact detail page.
3. For the dashboard (Refer to point 2 from before and after)
   1. Create a report with a graph
   2. Navigate to the dashboard
   3. Click on the Add widget button
   4. Filling details
      1. Name: <Enter name>
      2. Type: *Report Graph*
      3. Choose a graph: Select the graph name from the first step (3.1)
      4. Save
   5. Goto report and delete the report
   6. Clear the cache (`console c:c && console c:w`)
   7. Navigate the dashboard
